frontend/src/scenes/experiments/Experiment.tsx

```diff
@@ -571,6 +557,22 @@ export function Experiment(): JSX.Element {
                                             />
                                         </Col>
                                     </Row>
+                                    <Row className="secondary-metrics">
```

Comment:
Any reason to move this below? 🤔

Reply:
It was looking funny before I updated the UI for it but now everything is good so moving it back up ! 

<img width="765" alt="Screen Shot 2022-10-04 at 10 59 49 AM" src="https://user-images.githubusercontent.com/25164963/193854165-588deee4-662f-4950-9b59-e9478c9e5eba.png">